config/application.rb

```diff
@@ -53,7 +53,7 @@ def self.full(text)
 
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.2
+    config.load_defaults 6.1
```

Comment:
Turns out active_record.has_many_inversing isn't enabled until this is enabled (the config initializer doesn't take effect properly - https://github.com/rails/rails/issues/45683), and it also seems to be messing with our reply.rb item:

```rb
belongs_to :post, inverse_of: :replies, optional: false
```
somehow making the "Post must exist" validation stop executing. For now I'll keep the has_many_inversing config reverted and we can try figuring out why that's breaking the `optional: false` validation later.